app/src/components/Chat.tsx

```diff
@@ -173,7 +216,7 @@ export default function Chat({
         textareaRef={textareaRef}
         entryData={entryData}
         setEntryData={setEntryData}
-        waitingForResponse={waitingForResponse}
+        isStreaming={isStreaming}
```

Comment:
`isStreaming={isStreaming || waitingForResponse}`

one tiny change

Reply:
oh and the prop could do with a nicer name, maybe isDisabled instead of isStreaming